imports/plugins/core/catalog/server/no-meteor/schemas/schema.graphql

```diff
@@ -391,6 +391,7 @@ extend type Query {
     before: ConnectionCursor,
     first: ConnectionLimitInt,
     last: ConnectionLimitInt,
+    offset: ConnectionLimitInt,
```

Comment:
@cmbirk One more thing here. Can you add the missing param descriptions? You can paste in this, and adjust if you like.

```gql
"Return only results that come after this cursor. Use this with `first` to specify the number of results to return."
    after: ConnectionCursor,

    "Return only results that come before this cursor. Use this with `last` to specify the number of results to return."
    before: ConnectionCursor,

    "Return at most this many results. This parameter may be used with either `after` or `offset` parameters."
    first: ConnectionLimitInt,

    "Return at most this many results. This parameter may be used with the `before` parameter."
    last: ConnectionLimitInt,

    "Return only results that come after the Nth result. This parameter may be used with the `first` parameter."
    offset: ConnectionLimitInt,

    "Return results sorted in this order"
    sortOrder: SortOrder = desc,
```

Reply:
Descriptions look good to me. Resolved in cb4b8f770